src/components/ImgEditor/types.ts

```diff
@@ -13,4 +17,8 @@ export type Props<T extends FieldValues, K extends keyof T> = {
   accept: string[];
   classes?: Classes;
   aspect: [number, number];
+  /**
+   * Maximum image size in bytes
+   */
+  maxSize?: number;
```

Comment:
Adding `maxSize` prop makes `ImgEditor` tooltip at the bottom reflect the actual max image size (currently was hardcoded)